src/cmds/scripts/post_install.py

```diff
@@ -0,0 +1,463 @@
+print "Post install process started..."
```

Comment:
I really dislike the file name post_install.py because we already have a pbs_postinstall in the same directory. One approach would be to rename the file win_postinstall.py. Another approach would be to consolidate both postinstall scripts into a single python script that works everywhere.